robots/index.markdown

```diff
@@ -654,54 +654,54 @@ title: Robots
 		</div>
 		<div class="col-xs-12 col-sm-6 col-lg-3">
 			<div class="robots-single">
-				<div class="robots-image-wrapper" style="background-image: url('/assets/images/robots/fanuc/irmate200ib3l.png');">
+				<div class="robots-image-wrapper" style="background-image: url('/assets/images/robots/fanuc/lrmate200ib3l.png');">
 				</div>
 				<div class="robots-content">
-					<h2>irmate200ib3l</h2>
+					<h2>lrmate200ib3l</h2>
 					<h3>Fanuc</h3>
 					<a href="https://github.com/ros-industrial/fanuc" target="_blank">View package</a>
 				</div>
 			</div>
 		</div>
 		<div class="col-xs-12 col-sm-6 col-lg-3">
 			<div class="robots-single">
-				<div class="robots-image-wrapper" style="background-image: url('/assets/images/robots/fanuc/irmate200ib.png');">
+				<div class="robots-image-wrapper" style="background-image: url('/assets/images/robots/fanuc/lrmate200ib.png');">
 				</div>
 				<div class="robots-content">
-					<h2>irmate200ib</h2>
+					<h2>lrmate200ib</h2>
 					<h3>Fanuc</h3>
 					<a href="https://github.com/ros-industrial/fanuc" target="_blank">View package</a>
 				</div>
 			</div>
 		</div>
 		<div class="col-xs-12 col-sm-6 col-lg-3">
 			<div class="robots-single">
-				<div class="robots-image-wrapper" style="background-image: url('/assets/images/robots/fanuc/irmate200ic5h.png');">
+				<div class="robots-image-wrapper" style="background-image: url('/assets/images/robots/fanuc/lrmate200ic5h.png');">
 				</div>
 				<div class="robots-content">
-					<h2>irmate200ic5h</h2>
+					<h2>lrmate200ic5h</h2>
```

Comment:
```suggestion
					<h2>LR Mate 200<i>i</i>C/5h</h2>
```
https://www.fanucamerica.com/cmsmedia/datasheets/LR%20Mate%20200iC%20Series_10.pdf

Reply:
Thanks, I have changed all fanuc robots to their official model names.